backend/src/tasks/pools-updater.ts

```diff
@@ -136,18 +136,18 @@ class PoolsUpdater {
       axiosOptions.httpsAgent = new SocksProxyAgent(socksOptions);
     }
 
-    while(retry < 5) {
+    while(retry < config.MEMPOOL.EXTERNAL_MAX_RETRY) {
       try {
-        const data = await axios.get(path, axiosOptions);
-        if (data.statusText !== 'OK' || !data.data) {
+        const data: AxiosResponse = await axios.get(path, axiosOptions);
+        if (data.statusText === 'error' || !data.data) {
           throw new Error(`Could not fetch data from Github, Error: ${data.status}`);
         }
         return data.data;
       } catch (e) {
         logger.err('Could not connect to Github. Reason: '  + (e instanceof Error ? e.message : e));
         retry++;
       }
-      await setDelay();
+      await setDelay(config.MEMPOOL.EXTERNAL_RETRY_INTERVAL);
```

Comment:
Exposed this value to configuration as per discussed from https://github.com/mempool/mempool/pull/1649#discussion_r873354446